cfgov/v1/forms.py

```diff
@@ -278,10 +302,20 @@ class EnforcementActionsFilterForm(FilterableListForm):
     )
 
     def get_page_set(self):
-        query = self.generate_query()
-        return self.filterable_pages.filter(query).distinct().order_by(
-            '-initial_filing_date'
-        )
+        return EnforcementActionFilterablePagesDocumentSearch(
+            prefix=self.filterable_root,
+            topics=self.cleaned_data.get('topics'),
+            categories=self.cleaned_data.get('categories'),
+            authors=self.cleaned_data.get('authors'),
+            to_date=self.cleaned_data.get('to_date'),
+            from_date=self.cleaned_data.get('from_date'),
+            title=self.cleaned_data.get('title'),
+            statuses=self.cleaned_data.get('statuses')).search()
+
+        # query = self.generate_query()
+        # return self.filterable_pages.filter(query).distinct().order_by(
+        #     '-initial_filing_date'
+        # )
```

Comment:
```suggestion
```

Reply:
I think I'll reenable this section and use a flag to indicate which behavior needs to occur